src/components/Select/Select.vue

```diff
@@ -164,6 +168,12 @@ export default {
       default: '',
     },
     error: Boolean,
+
+    showPath: Boolean,
```

Comment:
I would just propose this prop to be called `showSubtitle` since it can be used for anything, not only paths of things. Right?